### PR TITLE
Application justification

### PR DESCRIPTION
**Pull request by mosaabtalib**
## Pull Request Checklist

- [ ] Does this PR includes changes from server side such as workflow, gateway, etc.?
- [x] Have you removed any unnecessary console.log?
- [x] Have you checked all variables, class names, function names, components, file names etc comply with our coding standards? 
- [x] Have you checked any spelling errors/issues?

### CODE REVIEW

**Comment by mosaabtalib** [comment]
UI mobile and UI desktop lang ito, to be followed ang logic

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "chart.js": "^3.7.1",
     "chartjs-plugin-datalabels": "^2.0.0",
     "core-js": "^3.6.5",
-    "dansalan-microservice-common": "^0.1.98",
+    "dansalan-microservice-common": "^0.1.228",
```
Comment (mosmdansalan) [inline package.json:32]: is this the correct version?

**Comment by mosmdansalan** [comment]
please resolve conflicts.

File: package.json (hunk 1 above)
Comment (mosaabtalib) [reply package.json]: nag npm install nalang ako ng latest sa dansalan

**Review by mosmdansalan — APPROVED** [review]

**Pull request merged**